Gruntfile.js

```diff
@@ -15,23 +15,49 @@ module.exports = function(grunt) {
         serverToken: '<%= secrets.serverToken %>'
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Yo',
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Hey',
         src: ['test/*.html']
       }
-    }
+    },
+
+    "postmark-templates": {
+      options: {
+        serverToken: '<%= secrets.serverToken %>',
+      },
+      build: {
+        name: "testing-template-node-js-" + new Date().valueOf(),
+        subject: "Testing grunt-postmark-templates",
+        // NOTE these are read from filesystem. globbing not supported
+        htmlFile: 'test/email.html',
+        textFile: 'test/email.txt',
+      }
+    },
+
+    "postmark-servers": {
```

Comment:
This is a neat idea. Can you move this to a separate PR? I think we should try to focus on sending and syncing templates with existing servers for now.